src/app/routes/utils/fetchPageData/index.test.js

```diff
@@ -5,15 +5,7 @@ import fetchPageData, { getUrl } from '.';
 const expectedBaseUrl = 'http://localhost';
 const requestedPathname = '/path/to/asset';
 const expectedUrl = `${expectedBaseUrl}${requestedPathname}.json`;
-const errorJson = ({ message }) =>
-  JSON.stringify(
-    {
-      event: 'data_fetch_error',
-      message,
-    },
-    null,
-    2,
-  );
+const DATA_FETCH_ERROR = 'data_fetch_error';
```

Comment:
Should we move this to` src/app/lib/logger.const.js` as well?

Reply:
It's already there! Good shout though, I will import it from the const file in my test, and prevent duplication 👍